mri/operators/fourier/non_cartesian.py

```diff
@@ -384,8 +384,15 @@ def adj_op(self, coeffs, *args):
         else:
             return self.impl.adj_op(coeffs, *args)
 
+    @property
+    def uses_sense(self):
+        """Return True if the Fourier Operator uses the SENSE method."""
+        try:
+            return self.impl.uses_sense
```

Comment:
How is this initialized? I dont see it anywhere in this PR.

Reply:
It's not initialized. By default its False for the base operator.